packages/flutter_tools/lib/src/localizations/localizations_utils.dart

```diff
@@ -442,6 +444,11 @@ class LocalizationOptions {
   /// Whether or not to format the generated files.
   final bool format;
 
+  /// The `format-args` argument.
+  ///
+  /// Arguments applied to `format` command.
```

Comment:
Replace with: "Arguments provided to `dart format` when formatting the generated files."

Reply:
Done.